graphql-server/Sources/App/GraphQL+HBResponseGenerator.swift

```diff
@@ -0,0 +1,13 @@
+import GraphQL
+import Foundation
+import Hummingbird
+
+extension GraphQLResult: HBResponseGenerator {
+    public func response(from request: HBRequest) throws -> HBResponse {
+        let encoded = try JSONEncoder().encode(self)
+        let buffer = request.allocator.buffer(data: encoded)
+        return HBResponse(status: .ok,
+                          headers: ["contnet-type" : "application/json; charset=utf-8"],
```

Comment:
`content-type` not `contnet-type`.

Reply:
You could also do this with. 
```swift
extension GraphQLResult: HBResponseEncodable {}
```
Extending a `Codable` object to conform to `HBResponseEncodable` does the same. `HBResponseGenerator` is really for anything that generates a response not using `Codable`. I should make this clearer in the docs

Reply:
> `content-type` not `contnet-type`.

Done

> ```swift
> extension GraphQLResult: HBResponseEncodable {}
> ```

VERY good to know! Thank you. And done.